GH-523: tx confirmation: DON'T MERGE, fast testing

bertllll opened the pull request.

bertllll committed:
GH-478: some undone pieces for chain_name solved + all blockchain_service_url added

bertllll committed:
GH-478: making an order in node_configurator_standard; it irked me many days - and plus formatting

bertllll committed:
GH-478: migration can be configured; one centric point to migrate was choosen in configurator_standard

bertllll committed:
GH-478: formatting

bertllll committed:
GH-478: I have neighborhood mode tooling for persistent config and 'configuration' command

bertllll committed:
GH-478: DbInitializer's configuration finalized; chain_id became redundand and is gone

bertllll committed:
GH-478: now also the current neighborhood mode is set into the database

bertllll committed:
GH-478: blockchain_service_url can be read from the db if not specified elsewhere

bertllll committed:
GH-478: formatting

bertllll committed:
GH-478: finishing stuff but I will probably need to handle an issue with tools for dump config in integration tests are not flexible enough

bertllll committed:
GH-478: back-up before trying to merge with some handy adjustment for integration tests from GH-305

bertllll committed:
GH-478: integration tests are complete; now I anticipate just refactoring on

bertllll committed:
GH-478: finishing merge with master

bertllll committed:
GH-478: changes shouting from multinode tests; little refactoring in run_modes

bertllll committed:
GH-478: trying to fix verify_bill_payment

bertllll committed:
GH-478: another adjustment in verify_bill_payment + renaming some stuff

bertllll committed:
GH-478: you change one thing's name and other module stand as if nothing happened; grrr

bertllll committed:
GH-478: merge between unprivileged and priveleged config fixed for blockchain_service_url; some small changes for BlockchainBridgeClandastine and verify_bill_payments()

bertllll committed:
GH-478: correction after auto-review

bertllll committed:
GH-473: mac specific test, detail

bertllll committed:
GH-503: compiling; unimplementeds() ahead of me

bertllll committed:
GH-503: merging GH-473 to be equel - it's a starting point of this one

bertllll committed:
GH-503: interim commit

bertllll committed:
Merge branch 'GH-473' into GH-503

bertllll committed:
GH-503: interim commit

bertllll committed:
GH-503: methods for transactings (the tools) are mockable

bertllll committed:
GH-503: before trying to create a reference from one field to another again

bertllll committed:
GH-503: all unimplemented are gone; next I should repair the tests of diferent signatures for each chain

bertllll committed:
GH-503: corrected error processing to be less ugly

bertllll committed:
GH-503: interim commit

bertllll committed:
Merge branch 'master' into GH-503

bertllll committed:
GH-503: deep automated tests for testnets as a proof of the signing process reliability

bertllll committed:
GH-503: unit tests seem complete

bertllll committed:
GH-503: refactored a bit; now a moment before trying to fix issue with unrecognized code from the library

bertllll committed:
GH-503: all tests should be ok...we can do refactoring if appropriate

bertllll committed:
GH-503: test suit saturated; all clean; first try in Actions

bertllll committed:
GH-478: right after aided merge; fixing bugs in the terrain next

bertllll committed:
GH-478: fixing broken code after the guided merge

bertllll committed:
GH-478: windows specific integration test adjusted

bertllll committed:
GH-478: minor fixes

bertllll committed:
GH-503-merged-with-478: the merge seems okay

bertllll committed:
GH-503-merged-with-478: formatting

bertllll committed:
GH-523: I've built up a solid test structure; standing before an implementation of a self directed message to BlockchainBridge to start the confirmation process

bertllll committed:
GH-523: I think I want BlockchainInterface to be a full mock too in the large test; so now before the change

bertllll committed:
GH-523: a couple of more functions for dealing with pending txs, mainly checking the transaction status remotely - yet now just mocked and then reasoning about the result

bertllll committed:
GH-523: merging master in

bertllll committed:
GH-523: before I'm gonna prepare Accountant for managing canceled and confirmed txs in his books

bertllll committed:
GH-523: a few more functions and tests; some refactoring for code from the previous day

bertllll committed:
GH-523: finished real impls of db changers for cancellations and confirmations

bertllll committed:
GH-523: merging changes for Rust 1.57 from master in here

bertllll committed:
GH-523: let's call the first stage done; I covered some more corner cases

bertllll committed:
GH-523: Good start to the rethought design; stable construction but pieces missing - tests patiently redriven for this changed situation

bertllll committed:
GH-523: the spinal test is repaired and passing but it should take a few more parameters to be complete; otherwise preparing for focusing on tests of smaller resolution tomorrow

bertllll committed:
GH-523: outer part of a payment backup works now

bertllll committed:
GH-523: interim commit

bertllll committed:
GH-523: backup - will continue with Payment may become PaymentWithMetadata

bertllll committed:
GH-523: fixing a few more tests inherited from the first version of the design

bertllll committed:
GH-523: minor stuff on tests, enhancing the code before moving on

bertllll committed:
GH-523: a few more situations tested

bertllll committed:
GH-492: scan_for_payables() seems fixed already

FinsaasGH committed:
GH-492: Merged in from my forked repo

bertllll committed:
GH-492: repairing a couple of tests; even very old ones which lied to test scanning in intervals; also one from previous try for this card. The part for payabales should be okay again

bertllll committed:
GH-492: merging

bertllll committed:
GH-523: beginning the transformation to a batch system (attempt 3); backup; before trying to get more tools - scanners as objects?

bertllll committed:
GH-523: master merged in

bertllll committed:
GH-523: a few new tools and tests restored

bertllll committed:
GH-523: setting up more tests properly, happy path is almost done

bertllll committed:
GH-523: a few more tests repaired

bertllll committed:
GH-523: a few more tests, a big portion in payable_dao

bertllll committed:
GH-523: more new tests, fixing DAOs, also repairing the big integration test, bud unfinished; plus potential issue with TransactionId discovered

bertllll committed:
GH-523: tests fixed; I can add some more functions now

bertllll committed:
GH-523: tracking number of confirmation attempts is enabled

bertllll committed:
GH-523: transaction_confirmed() and transaction_canceled() repaired + new mark_failure()

bertllll committed:
GH-523: small interim commit

bertllll committed:
GH-523: nonce added as a parameter in the backup

bertllll committed:
GH-523: pending payment time-out added

bertllll committed:
GH-523: more complex tx cancellation deffered for when we're ready; adding ignorring pending_txs with error marks

bertllll committed:
GH-523: main test in migrations is done; the toolset for writing migrations broadened a bit

bertllll committed:
GH-523: adding or modyfing less important tests; fixing details

bertllll committed:
GH-523: merging master

bertllll committed:
GH-523: fixing platform dependent error message in a test

bertllll committed:
GH-523: a small fix in a multinode test

bertllll committed:
GH-523: formatting

bertllll committed:
GH-523: one serious mistake with status interpretation and some unrelated details

bertllll committed:
GH-523: trying to turn the course of annoyance with the one test

bertllll committed:
GH-523-fast-testing

bertllll committed:
GH-523: merging in master

bertllll committed:
GH-523: formatting

bertllll committed:
GH-523-fast-testing: merging in master

bertllll committed:
GH-523: adjusting a test

bertllll committed:
GH-523: adjusting a test

bertllll committed:
GH-523: adjusting the same test

bertllll committed:
GH-523: adjusting the same test - a makeshift before GH-485 is played

bertllll committed:
GH-523-fast-testing: test adjustment

bertllll committed:
GH-523: fixing the same test again

bertllll committed:
GH-523: ignoring verify_bill_payment: will figure out the relation later

bertllll committed:
GH-523: adjusting default pending payment scan interval; was wrongly initialized in BootstrapperConfig

bertllll committed:
GH-523-fast-testing: fixes because the pending interval wrongly set

kauri-hero commented:
@bertllll in the final PR for this, could you please delete the following image files, which are obsolete and aren't attached to any code references.
https://github.com/MASQ-Project/Node/blob/master/MASQ_Logo_Gold.png 
https://github.com/MASQ-Project/Node/blob/master/MASQ_Logo_Stacked_Gold%403x.png

kauri-hero committed:
update to ratepack for testing Phase 1

interim changes before GH-523 merged into master

kauri-hero committed:
Update to scan and payment curve values for Phase1

Adjust payment curves and scan timing for GEMINI Phase1, prior to GH-523 merge to master

kauri-hero committed:
Merge branch 'master' into GH-523-fast-testing

kauri-hero committed:
conflict symbols missed

deleted conflict notation

kauri-hero committed:
delimiter

silly Kauri...

kauri-hero committed:
duplicate reference

kauri-hero committed:
delimiter error #2

kauri-hero committed:
formatted function wrong

kauri-hero committed:
missing semicolon

kauri-hero committed:
another delimiter snag
